torch/_inductor/decomposition.py

```diff
@@ -145,6 +145,113 @@ def addmm(input, mat1, mat2, *, beta=1, alpha=1):
     else:
         return NotImplemented  # go directly to lowering
 
+import time
+def bench(f, name=None, iters=100, warmup=5, display=True):
+    for _ in range(warmup):
+        f()
+    torch.cuda.synchronize()
+    begin = time.time()
```

Comment:
Maybe consider `torch.cuda.Event()` for timing?  (I'm not honestly sure how much it matters...)